includes/Modules/Analytics/Tag_Interface.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Tag_Interface
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics;
+
+/**
+ * Interface for an Analytics tag.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+interface Tag_Interface {
```

Comment:
+1 for adding this interface. While quite specific, it helps clarifying what an Analytics tag class implementation needs.